src/app/screen-snippet-handler.ts

```diff
@@ -381,7 +385,6 @@ class ScreenSnippet {
           `screen-snippet-handler: close window failed with error: ${error}!`,
         );
       }
-
       webContents?.focus();
```

Comment:
Will this be applied to close case while close is also has this issue?
I bet we also need focused window here